packages/core/useEyeDropper/index.ts

```diff
@@ -1,14 +1,28 @@
 import { ref } from 'vue-demi'
 
-export function useEyeDropper() {
+type WindowEyeDropper = Window & typeof globalThis & {
```

Comment:
Why don't we just type `EyeDropper` and also make it available in the return type?

Reply:
like this?
```typescript
export interface EyeDropper {
  new(): {
    open: () => Promise<{ sRGBHex: string }>
    [Symbol.toStringTag]: 'EyeDropper'
  }
}
```

Reply:
Yes